posts/2024-07-26-actuated-for-gitlab.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Secure CI for GitLab with Actuated.
```

Comment:
`Secure CI for GitLab with Actuated.`

=>

`Secure your GitLab jobs with microVMs and Actuated`